src/fal/packages/isolated_runner.py

```diff
@@ -74,7 +91,7 @@ def main() -> None:
         message = "=" * 60
         message += "\n" * 3
         message += "Debug mode successfully activated. You can start your debugging session with the following command:\n"
-        message += f"    $ {sys.executable} {os.path.abspath(__file__)} --with-pdb {options.listen_at}"
+        message += f"    $ {_get_shell_bootstrap()} {sys.executable} {os.path.abspath(__file__)} --with-pdb {options.listen_at}"
```

Comment:
```suggestion
        message += f"    $ {_get_shell_bootstrap()}\\\n     {sys.executable} {os.path.abspath(__file__)} --with-pdb {options.listen_at}"
```

to give it in 2 lines (easy to understand?)

Reply:
Much better indeed!